service/src/test/java/bio/terra/cbas/service/TestRunSetsService.java

```diff
@@ -243,6 +247,9 @@ void init() {
     uuidSource = mock(UuidSource.class);
     objectMapper = mock(ObjectMapper.class);
     cbasContextConfiguration = mock(CbasContextConfiguration.class);
+    bardService = mock(BardService.class);
+    bardServerConfiguration = mock(BardServerConfiguration.class);
+    when(bardServerConfiguration.enabled()).thenReturn(true);
```

Comment:
If we pushed this check down into the bard service itself, we could have a service-level unit test to verify that it calls the API when enabled and doesn't when it's disabled